src/components/ExamplesDropdown.tsx

```diff
@@ -83,6 +87,12 @@ export function ExamplesDropdown({
             <AlertDialogAction
               onClick={() => {
                 if (selectedExample) {
+                  if (config.posthog.apiKey && config.posthog.host) {
```

Comment:
You don't need these conditionals - if it isn't configured it just adds it to a local array, similar to how google analytics works.